doc/conf.py

```diff
@@ -17,7 +17,7 @@
 # built documents.
 #
 # The short X.Y version.
-version = '3.0'
+version = '3.0.1dev1'
```

Comment:
Should this also be ``3.0.1dev2``, like in ``setup.py``?

Reply:
Yes. But the most important part is, that set this to 3.0.1 when we are done! ;-)
Will do.

Reply:
Hm, lets see if this appears, when i push on the branch-3.0

Reply:
done in 656fe468